### ISSUE TITLE
How to choose the final results

### ISSUE THREAD

**Issue by dsy412610** [fchollet/keras#2077]
After numbers of iterations, validation accuracy leveled off and fluctuated. So I don't know how to choose the final results. How?

**Comment by tboquet** [comment]
You could stop the training when the validation loss is not decreasing anymore. You could take a look at the [faq](http://keras.io/faq/#how-can-i-interrupt-training-when-the-validation-loss-isnt-decreasing-anymore) in the documentation.

**Comment by dsy412610** [comment]
Thanks! But how should I set the value of patience, with experience or some rules？
